### PR TITLE
Adding another stanza to the JSON that external partner has requested.

### PR DESCRIPTION
**Pull request by alxdarksage**
For some reason this will allow them to display the correct organization when a user first signs in to their system.

Also added a callback to Columbia when an appointment record is being stored with our system, to look up details about the appointment location.

### CODE REVIEW

File: src/test/java/org/sagebionetworks/bridge/spring/controllers/CRCControllerTest.java [hunk 1]
```diff
@@ -443,12 +462,65 @@ public void postAppointmentUpdated() throws Exception {
...
+
+        Appointment appointment = new Appointment();
+        addAppointmentParticipantComponent(appointment, PATIENT_NS + USER_ID);
+        
+        String json = FHIR_CONTEXT.newJsonParser().encodeResourceToString(appointment);
+        mockRequestBody(mockRequest, json);
+        
+        controller.postAppointment();
+        verify(mockHealthDataService).submitHealthData(any(), any(), any());        
     }
```
Comment (DwayneJengSage) [inline src/test/java/org/sagebionetworks/bridge/spring/controllers/CRCControllerTest.java]: Should we also add a check that we never call HTTP get()?

File: src/test/java/org/sagebionetworks/bridge/spring/controllers/CRCControllerTest.java [hunk 2]
```diff
@@ -418,7 +431,11 @@ public void postAppointmentCreated() throws Exception {
         assertEquals(healthData.getAppVersion(), "v1");
         assertEquals(healthData.getCreatedOn(), TIMESTAMP);
         assertEquals(healthData.getMetadata().toString(), "{\"type\":\""+APPOINTMENT_REPORT+"\"}");
-        assertEquals(healthData.getData().toString(), json);
+        assertEquals(healthData.getData().toString(), TestUtils.createJson("{'resourceType':'Appointment',"
+                +"'participant':[{'actor':{'reference':'Location/some-other-id','telecom':"
+                +"[{'system':'phone','value':'1231231235','use':'work'}],'address':{'line':"
+                +"['123 east 165'],'city':'New York','state':'NY','postalCode':'10021'}}},"
+                +"{'actor':{'reference':'Patient/userId'}}]}"));
```
Comment (DwayneJengSage) [inline src/test/java/org/sagebionetworks/bridge/spring/controllers/CRCControllerTest.java:438]: Should we also verify the URL for get location?

**Review by DwayneJengSage — APPROVED** [review]
Looks good. A few notes about tests. I'll leave it to your judgement whether it's worth incorporating these comments.

**Comment by alxdarksage** [comment]
These are good ideas, thanks for the review, both have been added.

**Pull request merged**